syndeo-template/pom.xml

```diff
@@ -234,12 +234,12 @@
             <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
             <version>${beam.snapshot.version}</version>
         </dependency>
-        <dependency>
+        <!-- <dependency> -->
             <!-- Direct runner included for local development and testing. -->
-            <groupId>org.apache.beam</groupId>
+            <!-- <groupId>org.apache.beam</groupId>
```

Comment:
Not sure if we should remove this? Even though I've never run a pipeline locally, but I don't know if this use case is completely irrelevant

Reply:
I don't think anyone runs syndeo on direct runner, not even the tests do that.

Reply:
If there is a valid use case, I can add it back.